options/locale/locale_en-US.ini

```diff
@@ -1815,8 +1815,8 @@ milestones.edit_success = Milestone "%s" has been updated.
 milestones.deletion = Delete Milestone
 milestones.deletion_desc = Deleting a milestone removes it from all related issues. Continue?
 milestones.deletion_success = The milestone has been deleted.
-milestones.filter_sort.closest_due_date = Closest due date
-milestones.filter_sort.furthest_due_date = Furthest due date
+milestones.filter_sort.closest_due_date = Latest
+milestones.filter_sort.furthest_due_date = Earliest
```

Comment:
Please be careful, I think otbutz‘s idea is `Closest due date -> Earliest due date`, not ` -> Latest`. You ignored "due data" and got a wrong match.

Reply:
My apologies. Thanks for the tip.